### PR TITLE
installation: latest base package

### LINKED ISSUE #3572: docs: replace manage.py with instance command
- [ ] Since Invenio-Base has been released (https://github.com/inveniosoftware/invenio-base/releases/tag/v1.0.0a4), usage of `manage.py` needs to be amended (https://github.com/inveniosoftware/invenio/search?utf8=%E2%9C%93&q=manage.py).
- [ ] Invenio now runs also on SQLite and PostgreSQL - README.rst: ... requires MySQL database server ... needs an update too

### PR DESCRIPTION
**Pull request by tiborsimko**
* BETTER Amends installation scripts and documentation to use the
  recently released `Invenio-Base` package where `manage.py` was removed
  in profit of `setup.py`. (closes #3572)

* Amends README file to mention PostgreSQL and SQLite alongsite MySQL in
  the list of supported databases.

Signed-off-by: Tibor Simko <tibor.simko@cern.ch>

### CODE REVIEW

**Comment by tiborsimko** [comment]
(looking at record loading troubles...)

**Comment by tiborsimko** [comment]
(loading troubles fixed)

File: scripts/install.sh [hunk 1]
```diff
@@ -104,9 +104,9 @@ set -o errexit
 set -o nounset
 
 # sphinxdoc-install-invenio-full-begin
-# FIXME needed for invenio-search
+# FIXME the next command needed until https://github.com/inveniosoftware/invenio-search/pull/34 is released on PyPI
```
Comment (jirikuncar) [inline scripts/install.sh]: it has been released

Comment (tiborsimko) [inline scripts/install.sh]: Done.  I have also updated to the latest `invenio-records-ui`, `invenio-theme` and friends.

**Pull request merged**